internals/secrethub/service_init_test.go

```diff
@@ -0,0 +1,244 @@
+package secrethub
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/secrethub/secrethub-cli/internals/cli/filemode"
+	"github.com/secrethub/secrethub-cli/internals/cli/ui/fakeui"
+	"github.com/secrethub/secrethub-go/internals/api"
+	"github.com/secrethub/secrethub-go/internals/assert"
+	"github.com/secrethub/secrethub-go/internals/errio"
+	"github.com/secrethub/secrethub-go/pkg/secrethub"
+	"github.com/secrethub/secrethub-go/pkg/secrethub/credentials"
+	"github.com/secrethub/secrethub-go/pkg/secrethub/fakeclient"
+)
+
+func TestServiceInitCommand_Run(t *testing.T) {
+	keyCreator := credentials.CreateKey()
+	_ = keyCreator.Create()
+	val, _ := keyCreator.Export()
```

Comment:
I suggest renaming this variable to `exportedCredential` to make it more clear what it contains.

Reply:
Nice catch. Fixed